tests/AbstractSinglePathProcessingTaskTest.php

```diff
@@ -189,7 +189,7 @@ public function testHandlesParallelProcess(): void {
     $this->process
       ->expects($this->exactly(2))
       ->method('isRunning')
-      ->willReturnOnConsecutiveCalls([TRUE, FALSE]);
+      ->willReturnOnConsecutiveCalls(TRUE, FALSE);
```

Comment:
isRunning() method should be returning boolean, but this was mocking it so that it would return array of those booleans. In older version is seemed to have worked, but in newer version of Symfony it has return type defined so it failed:

![image](https://user-images.githubusercontent.com/492375/229125348-e10d9a97-e905-44be-a48c-a3cbbb54cbc9.png)